migrations/20250408094238_members-additional-col.js

```diff
@@ -0,0 +1,11 @@
+export const up = async (knex) => {
+  await knex.schema.alterTable('members', (def) => {
+    def.string('role', 20).notNullable().defaultTo('None')
+  })
+}
+
+export const down = async (knex) => {
+  await knex.schema.alterTable('members', (def) => {
+    def.dropColumn('role')
+  })
+}
```

Comment:
I think we need to think about the db structure a bit and the extensibility of the identity service. We are going to have different use cases of sequence with different role names. My temptation is to create a table with the allowed role names in that we then foreign key to. That way we can set the allowed roles based on the values in the db. The helm chart could then do a `psql` statement to ionsert the necessary roles. 

The service would then need to query the database to know the allowed roles, think the API spec would have to then either be monkey patched or we allow all strings.